CinemaTicketSalesSystem/CinemaTicketSalesBusinessLogic/CRUD/GenreService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using ApplicationDbMovies.Contexts;
+using ApplicationDbMovies.Models;
+using AutoMapper;
+using CinemaTicketSalesBusinessLogic.Interfaces;
+using CinemaTicketSalesBusinessLogic.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CinemaTicketSalesBusinessLogic.CRUD
+{
+    public class GenreService : IGenreService
+    {
+        private readonly ApplicationDbContext _db;
+        private readonly IMapper _mapper;
+
+        public GenreService(ApplicationDbContext db, IMapper mapper)
+        {
+            _db = db;
+            _mapper = mapper;
+        }
+
+        /// <summary>
+        /// This method is mapping from Genre model to GenresModel
+        /// </summary>
+        public IEnumerable<GenresModel> GetGenresModel()
+        {
+            var genres = _db.Genres.ToList();
```

Comment:
better to do is like: 
var genres = _db.Genres.ToList();
var genreModels = _mapper.Map<IEnumerable<Genre>, IEnumerable<GenreModel>>(genres);
return genresModels;